views/components/table.html

```diff
@@ -0,0 +1,25 @@
+{% macro table(tableData) %}
+<article class="base--article" itemscope itemtype="http://schema.org/Table">
+  <header class="base--header">
+    <h1 class="base--h1" itemprop="name">{{tableData.name}}</h1>
+  </header>
+  <table class="base--table">
+    <thead class="base--thead">
+      <tr class="base--tr">
+        {% for header in tableData.data.headers %}
```

Comment:
The header information should live in `data.headers`, `data` coming from the passed in `data` in the macro. This should read `{% for header in data.headers %}`